pkg/sign/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package sign provides functionality to sign fleet config using a ECDH private key
```

Comment:
`Package sign provides functionality to sign fleet configs using an ECDH private key.`
(Add period, a->an, config->configs).

Reply:
updated